site/src/styles/site.css

```
@@ -0,0 +1,158 @@
+  --line-soft: #1f262e;
+  --text: #f3f7f5;
+  --muted: #aeb9bf;
+  --quiet: #6f7d85;
+  --mint: #a6f4d3;
+  --mint-strong: #73e6b6;
+  --coral: #ff8f70;
+  --shadow: 0 24px 80px rgb(2 10 16 / 34%);
+  --radius-lg: 24px;
+  --radius-md: 16px;
+  --content: 1180px;
+  font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)